user_manager/views.py

```diff
@@ -16,6 +18,47 @@ def get(self, request):
         return Response(serializer.data)
 
 
+class CheckValidCredentials(APIView):
+    permission_classes = [AllowAny]
+
+    def get(self, request):
+        if request.user.is_authenticated:
+            return Response(status=status.HTTP_200_OK)
+        else:
+            return Response(status=status.HTTP_401_UNAUTHORIZED)
+
+
+class UserLoginView(APIView):
+    permission_classes = [AllowAny]
```

Comment:
What does the `AllowAny` permission_classes actually allow? Asking as it looks a little scary 😬

Reply:
Hahaha, that just makes it so that any caller can send a request to that endpoint (i.e., they don't need to include any credentials). We need this so that an unauthenticated user can login.